### ISSUE TITLE
unable to access keybordshortcut

### ISSUE THREAD

**Issue by abhijitpatil181171** [iTwin/itwinjs-core#5921]
const handleIModelAppInit = () => {
    UiFramework.keyboardShortcuts.loadShortcuts(
      AccuDrawKeyboardShortcuts.getDefaultShortcuts()
    );
  };
  In above code i am getting error at keybordshortcut saying Property 'keyboardShortcuts' does not exist on type 'typeof UiFramework'.ts i have imported uiframework also then what is issue?

* * *

_⚠ Do not edit this section. It is required for imodeljs.github.io ➟ GitHub issue linking_

*   LINK: UiFramework---appui-react-link-to-docs
*   TAG: imodeljs-docs-question
*   SOURCE: https://www.itwinjs.org/reference/appui-react/utilities/uiframework/

**Comment by YashodipD** [comment]
Hi @abhijitpatil181171 , Here is my wildest possible guess. If you are using Viewer component from @itwin/web-viewer-react. On viewer component there is prop called **toolAdmin** , pass an instance of `new FrameworkToolAdmin()` to it. 

```
<Viewer
        iTwinId={iTwinId ?? ""}
        iModelId={iModelId ?? ""}
        authClient={authClient}
        uiProviders={uiProviders}
        enablePerformanceMonitors={false}
        onIModelConnected={handleIModelConnected}
        theme={process.env.THEME ?? "dark"}
        **toolAdmin={new FrameworkToolAdmin()}**
        onIModelAppInit={handleAppInit}
        viewportOptions={viewportOptions}
      />
```

If this does not help, other experts might be help you.

**Comment by raplemie** [comment]
Hi, which version of the packages are you using ? `keyboardShortcuts` was introduced in 3.7.0.

**Comment by GerardasB** [comment]
Closing this issue due to inactivity. As suggested above, `UiFramework.keyboardShortcuts` is available from 3.7.0 https://www.itwinjs.org/changehistory/3.7.0/#static-manager-classes
